### ISSUE TITLE
Problem with training 

### ISSUE THREAD

**Issue by ghost** [Vectorized/VoxNet-Tensorflow#1]
Thank you for the implementation.

I followed the code voxnet_train.py. However, after several saved checkpoints 100, the percentage of training and test are still low around 0.14. It is not like you described " After 20 minutes of training from scratch on a Nvidia Titan X, it is able to get 86%+ test accuracy."

Could you verify again the voxnet_train.py?

Thank you very much

**Comment by Vectorized** [comment]
Sorry for the late reply.

Which version of Tensorflow are you using? 
I'm using 1.9.0 GPU version and have tested it on another machine.  
Not sure if a newer Tensorflow version will break things. 

If you have modified or re-implemented it, do check the learning rates and the layers are similar.

Check that the cross-entropy loss is decreasing as you train. It should converge to +/- 0.2.
The learning rate begins at 0.001 is slowly lowered as training goes by using exponential decay.
